lib/absinthe/subscription.ex

```diff
@@ -143,16 +143,14 @@ defmodule Absinthe.Subscription do
   def subscribe(pubsub, field_key, doc_id, doc) do
     registry = pubsub |> registry_name
 
-    doc_value = {
-      doc_id,
-      %{
-        initial_phases: PipelineSerializer.pack(doc.initial_phases),
-        source: doc.source
-      }
+    doc_value = %{
+      initial_phases: PipelineSerializer.pack(doc.initial_phases),
+      source: doc.source
     }
 
     pdict_add_field(doc_id, field_key)
-    {:ok, _} = Registry.register(registry, field_key, doc_value)
+    {:ok, _} = Registry.register(registry, field_key, doc_id)
+    {:ok, _} = Registry.register(registry, doc_id, doc_value)
```

Comment:
Is the `doc_id` guaranteed to be unique per `field_key`? If not, we may want to key it here too:

```suggestion
    {:ok, _} = Registry.register(registry, [field_key | doc_id], doc_value)
```

Reply:
A document can subscribe to only one field, so I do not think we need to include the `field_key` here.